source/playgrounds.txt

```diff
@@ -261,6 +261,7 @@ Consideration for Authentication
 .. toctree::
    :titlesonly:
 
+   /enable-autocomplete
```

Comment:
From a TOC ordering persepctive, I'd suggest maybe putting this after "Run Aggregation Pipelines". I think we want to keep critical tasks toward the top of the list.

Reply:
10-4, my thinking was for best user experience, you would want to enable the Autocomplete before getting started with any work, but this makes sense.